src/main/java/frc/robot/subsystems/Led.java

```diff
@@ -134,10 +141,26 @@ public void handleLeds(LedSection section) {
                 ledBuffer.setRGB(ledBuffer.getLength() - ledPosition - 1, 0, 255, 0);
                 Logger.getInstance().recordOutput("IndicateLEDs", "Spinning");
                 break;
+            case BalanceLeds:
+                if (autoBalanceState.get(false)) {
```

Comment:
I think your logic is reversed - if it's balanced, then it'll flash, but if it's not balanced it'll be green?

Reply:
Not quite right - now you changed it so that if the data doesn't exist yet, it'll assume it's balanced.